### DISCUSSION TITLE
spaCy Project: Part-of-speech Tagging & Dependency Parsing

### DISCUSSION THREAD

**Discussion by kanayer** [explosion/spaCy#9798] [Help: Coding & Implementations] [answered]
Can the project located [here](https://github.com/explosion/projects/tree/v3/pipelines/tagger_parser_ud) be used to train the tagger & parser model on my own dataset? I have my own dataset in the `conllu ` format and it uses UD tags, but it is not one of the official UD datasets. Does it necessarily need to be from the UD GitHub repo?

**Comment by polm** [comment, marked as answer]
If your data is in the same format it should still work, you just need to adjust the project assets and preparation step.
